### PR TITLE
Kubernetes deployments for stub api

### PR DESCRIPTION
**Pull request by m-doughty**
### Name of the story

These are changes from the morning automatic batch catch-up with Dilu.

### Acceptance Criteria

- [x] Can now deploy stub API to Kubernetes;
- [x] Ingress is set up for stub API;
- [x] LetsEncrypt is set up for stub API.

### Pre-flight Checklist

- [ ] Tests
- [ ] DUXD review approval
- [ ] Accessibility Reviewed
- [ ] Product owner demo

### Pre-merge Checklist

- [ ] Branch test approved

### CODE REVIEW

**Comment by m-doughty** [comment]
To see this working in AKS:

https://document-manager.uksouth.cloudapp.azure.com/documents/con10101010

File: medicines/document-manager/stub-api/README.md [hunk 1]
```diff
@@ -25,6 +25,50 @@ $ docker run -p 8080:8080 -it --rm stub-document-manager-api
...
+$ helm repo update
+
+$ helm install api-stub stable/nginx-ingress \
+  --set controller.replicaCount=2 \
+  --set controller.nodeSelector."beta\.kubernetes\.io/os"=linux \
+  --set defaultBackend.nodeSelector."beta\.kubernetes\.io/os"=linux \
+  --set controller.service.loadBalancerIP=$YOUR_IP_ADDRESS \
+  --set controller.service.externalTrafficPolicy=Local
+
+$ kubectl apply --validate=false -f https://raw.githubusercontent.com/jetstack/cert-manager/release-0.12/deploy/manifests/00-crds.yaml
```
Comment (pataruco) [inline medicines/document-manager/stub-api/README.md]: if we are using this lock version for signing a certificate, would it be possible to move this manifest to our repo and reference?

File: medicines/document-manager/stub-api/README.md [hunk 2]
```diff
@@ -25,6 +25,50 @@ $ docker run -p 8080:8080 -it --rm stub-document-manager-api
...
+$ helm install cert-manager jetstack/cert-manager
+```
+
+Then you need to build the Docker image and push it to your container registry:
+
+```bash
+$ export STUB_IMAGE=container-registry.yourdomain.com/stub-api:1.0.1
```
Comment (pataruco) [inline medicines/document-manager/stub-api/README.md]: We can create on terraform a container registry for this

Comment (m-doughty) [reply medicines/document-manager/stub-api/README.md]: Yeah this would be best.

File: medicines/document-manager/stub-api/README.md (hunk 1 above)
Comment (m-doughty) [reply medicines/document-manager/stub-api/README.md]: As discussed, due to the ephemeral nature of the stub and the fact that we don't intend to use Helm going forward for Kubernetes dependency management & templating, it's not really a high priority to introduce helm to the stub repo with a full dependency chain & `values.yaml`.

**Pull request merged**